docs/config/index.md

```diff
@@ -485,10 +485,10 @@ export default ({ command, mode }) => {
 
 ### build.lib
 
-- **Type:** `{ entry: string, name?: string, formats?: ('es' | 'cjs' | 'umd' | 'iife')[] }`
+- **Type:** `{ entry: string, name?: string, formats?: ('es' | 'cjs' | 'umd' | 'iife')[] }, fileName?: string`
```

Comment:
`fileName` should go inside the object here `{ ... }`

Reply:
> `fileName` should go inside the object here `{ ... }`

yes, I have made changes ,thanks

Reply:
Thanks! Theres is a missing space before the last `}`, would you also correct that?